JitsiConference.js

```diff
@@ -3663,3 +3669,62 @@ JitsiConference.prototype.lobbyApproveAccess = function(id) {
         this.room.getLobby().approveAccess(id);
     }
 };
+
+/**
+ * Returns <tt>true</tt> if AV Moderation support is enabled in the backend.
+ *
+ * @returns {boolean} whether AV Moderation is supported in the backend.
+ */
+JitsiConference.prototype.isAVModerationSupported = function() {
+    return Boolean(this.room && this.room.getAVModeration().isSupported());
+};
+
+/**
+ * Enables AV Moderation by moderators
+ * @param {MediaType} mediaType "audio" or "video"
+ */
+JitsiConference.prototype.enableAVModeration = function(mediaType) {
+    if (this.room && this.isModerator()
+        && (mediaType === MediaType.AUDIO || mediaType === MediaType.VIDEO)) {
+        this.room.getAVModeration().enable(true, mediaType);
+    } else {
+        logger.warn(`Failed to enable AM moderation, no room or not a moderator:${
```

Comment:
typo: AM instead of AV. Also, please make the message easier to understand without the code

Reply:
Fixed